1. Standard Python/Lists & Tuples/main.py

```diff
@@ -29,3 +29,9 @@
     print(tup[i])
 for item in tup:
     print(item)
+
+
+# Apply the same behaviour to all items of a list - here add 2 to each number
```

Comment:
For clarity, might want to mention that the results are stored in a new list, and the initial list is unchanged.